### PR TITLE
feat(apolo-dev): new workflow tag for apolo-dev workloads deployment

### PR DESCRIPTION
**Pull request by ESlam-Yousry**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
   deploy_dev:
     name: Deploy on dev
     needs: release
-    uses: neuro-inc/reuse/.github/workflows/deploy-dev.yaml@v24.8.0
+    uses: neuro-inc/reuse/.github/workflows/deploy-dev.yaml@v0.0.33
```
Comment (asvetlov) [inline .github/workflows/ci.yaml:53]: The version decreasing looks strange to me.
Please describe, why did you change the old naming conversion?

Comment (ESlam-Yousry) [reply .github/workflows/ci.yaml:53]: yeah, but check reuse pipeline versions i did use the old versions because there's some irrelevant versions
